addons/crm/static/src/js/crm_graph_forecast.js

```diff
@@ -0,0 +1,25 @@
+/** @odoo-module alias=crm.CrmGraphForecast **/
+import GraphView from 'web.GraphView';
+import viewRegistry from 'web.view_registry';
+
+/**
+ * Graph view to be used for a Forecast @see ForecastModelExtension
+ * requires:
+ * - context key `forecast_field` on a date field
+ * - special filter "Forecast" (which must set the `forecast_filter:1` context key)
+ */
+var GraphForecastView = GraphView.extend({
+    /**
+     * @override
+     */
+    _createSearchModel(params, extraExtensions = {}) {
+        Object.assign(extraExtensions, { Forecast: {} });
+        return this._super(params, extraExtensions);
+    },
+});
+
+viewRegistry.add('graph_forecast', GraphForecastView);
+
+export default {
+    GraphForecastView: GraphForecastView,
+};
```

Comment:
Why does this one export a mapping when the ForecastService one exported the object directly?